### PR TITLE
Add a Lookup Plugin to get the Checkmk Version of a Server

### PR DESCRIPTION
**Pull request by lgetwan**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

There are no lookup plugins at all

## What is the new behavior?

- With the lookup plugin, the user can query the server's version.

## Other information

- The full demo playbook now also uses this lookup

### CODE REVIEW

File: playbooks/demo/lookup.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  hosts: test
+  strategy: linear
+  gather_facts: false
+  vars_files:
+    - ../vars/auth.yml      # This vars file provides details about your site
+  tasks:
+
+    - name: "Get CMK version."
+      debug:
+        msg: "Version is {{ lookup('checkmk.general.version', server_url+'/'+site, validate_certs=False, automation_user=automation_user, automation_secret=automation_secret)}}"
```
Comment (robin-checkmk) [inline playbooks/demo/lookup.yml:12]: Jinja2 spacing should be improved, but then we hit the line length. I can take care of it later, have to look up the syntax again.

**Review by robin-checkmk — COMMENTED** [review]
I cannot get the demo playbook to work, Ansible complains about not finding the lookup module. But that might be a local issue on my machine.

File: plugins/lookup/version.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+EXAMPLES = """
+- name: We could read the file directly, but this shows output from command
+  ansible.builtin.debug:
+      msg: "CMK version: {{ lookup('checkmk.general.version', 'https://myserver/mysite', automation_user='automation', automation_secret='$SECRET'}}."
+"""
+
+RETURN = """
+  _list:
+    description:
+      - server CMK version
```
[the remark this answers is not in the document]
Comment (lgetwan) [reply plugins/lookup/version.py]: Corrected.

File: playbooks/demo/lookup.yml (hunk 1 above)
Comment (lgetwan) [reply playbooks/demo/lookup.yml]: I've put the lookup reply into a variable and used it in the msg.
That way, I was able to add line breaks to make the lookup call look nicer.

Comment (robin-checkmk) [reply playbooks/demo/lookup.yml]: Love it!

**Review by robin-checkmk — APPROVED** [review]

**Review by Max-checkmk — APPROVED** [review]
Cool stuff! Collection is growing! ;)

**Pull request merged**